crates/sovereign-sdk/rollup-interface/src/node/rpc/mod.rs

```diff
@@ -695,6 +698,80 @@ pub mod utils {
             deserializer.deserialize_str(HexStrVisitor(PhantomData))
         }
     }
+
+    /// Serialization and deserialization logic for `0x`-prefixed hex strings that are wrapped with Option.
```

Comment:
Use TxHex that already exists in the project if you can.

Reply:
Done https://github.com/chainwayxyz/citrea/pull/867/commits/4c78cd7ba2a84b67a8cd27d98d3a45c4b8fbd7f6